docs/documentation/94/resultset.md

```diff
@@ -16,4 +16,11 @@ The following must be considered when using the `ResultSet` interface:
 * You must close a `ResultSet` by calling `close()` once you have finished using
 	it.
 * Once you make another query with the `Statement` used to create a `ResultSet`,
-	the currently open `ResultSet` instance is closed automatically.
\ No newline at end of file
+	the currently open `ResultSet` instance is closed automatically.
+* When PreparedStatement API is used, `ResultSet` switches to binary mode after 
```

Comment:
I would suggest making this a bit more generic. Something to the effect of the result of calling getString for data types which are not strings is not guaranteed. While results will be logically equivalent, the format may be inconsistent based things like backend string representation vs. binary mode or even if a complex type (such as an array) has been converted into object form.

Reply:
Working on this。